app/screens/Account/surveys.js

```diff
@@ -87,10 +106,13 @@ class SurveysScreen extends Component {
   }
 }
 
-const styles = StyleSheet.create({
-  subtitle: {
-    fontSize: 17
-  }
+const mapStateToProps = state => ({
+  availableSurveys: selectAvailableSurveys(state),
+  remoteSurveys: selectRemoteSurveys(state)
 });
 
-export default SurveysScreen;
+export default connect(mapStateToProps, {
+  clearRemoteSurveys,
+  fetchRemoteSurvey,
+  listRemoteSurveys
```

Comment:
I'm not sure how to do this without being somewhat error-prone. These come from `actions/index.js` and are passed here as an object so that each will be bound to `dispatch` (they'll later be extracted from props). However, if one forgets to use the version passed in via `props`, no error will be raised and the action creator will still be executed **but not dispatched**.

Thoughts?